tests/scripts/thread-cert/v1_2_test_csl_transmission.py

```diff
@@ -71,6 +71,11 @@ def test(self):
         self.assertTrue(self.nodes[LEADER].ping(self.nodes[SSED_1].get_rloc()))
         self.simulator.go(5)
 
+        self.nodes[SSED_1].send_mac_data()
+        self.simulator.go(1)
+        self.nodes[SSED_1].send_mac_datarequest()
+        self.simulator.go(1)
+
```

Comment:
For SSED_1 topology config in above Line#48-#51, add `is_mtd: True` to use ot-cli-mtd and set `USE_MTD=1`  env.